### DISCUSSION TITLE
Add: Ability to tile dashboard wallpapers

### DISCUSSION THREAD

**Discussion by marcinbauer-ohf** [home-assistant/frontend#22512] [Dashboard] [answered]
## Description
This design addresses a proposal to add the ability to tile background images of views via the UI.

> [!NOTE]
> Detailed notes and previews can be found in the Figma file.

## Design & Notes
https://www.figma.com/design/HMPvPfk5K2BGbsB2imkrCI/HA-06?node-id=2001-1453&node-type=section&t=pHdBoFsyaqIndK3f-11

> [!TIP]
> To preview Figma files directly in Github please consider installing [Figink](https://chromewebstore.google.com/detail/figink/iaieddclhpjmcpigojanfjdmcmgonnac) Chrome extension

_- Home Assistant (Nabu Casa) Product Team_

**Comment by karwosts** [comment]
https://github.com/home-assistant/frontend/pull/22643

**Comment by marcinbauer-ohf** [comment, marked as answer]
https://github.com/home-assistant/frontend/discussions/22931
